### ISSUE TITLE
Got this error message "updating" an installed appimage

### ISSUE THREAD

**Issue by star-buck** [TheAssassin/AppImageLauncher#57]
Is this an AIL issue or the Opendesktop Appimage doesnt use update feature correctly?
![screenshot_20180705_211754](https://user-images.githubusercontent.com/840070/42343363-1ac68cb6-8099-11e8-9d1b-d8e1d626cd6e.png)

**Comment by TheAssassin** [comment]
IIRC openDesktop Store doesn't implement updating at the moment. I'd be happy to help you get started on this.

By the way, yesterday there's been an update to AppImageUpdate and AppImageLauncher's update tool to actually display the error messages (see https://github.com/TheAssassin/AppImageLauncher/issues/54). With the latest version, you can now see what exactly made the update check fail.

**Comment by star-buck** [comment]
So with new ail version it states if the Appimage doesnt seem to support updating, it didnt find any new updates or if ail were not able to check for updates?
Best would be to never say "Error" in any case, but just an neutrally information message :)

**Comment by TheAssassin** [comment]
You're absolutely right. Missing update information shouldn't be displayed as error.

**Comment by Silicomancer** [comment]
A little OT: I tried to find information on how the update mechanism works and on how to add update-information to my AppImage (especially using `linuxdeployqt`, without CI). I didn't find anything useful. Is there a guide somewhere?
